### DISCUSSION TITLE
Regarding the mouseover event penetration issue.

### DISCUSSION THREAD

**Discussion by zszq** [pixijs/pixijs#9857] [Q&A]
I drew a shape on the canvas and added a mouseover event to it. Then I positioned a div element to cover this shape. When I move the mouse over this div element, the mouseover event is triggered. How can I prevent this mouseover event from being triggered?

**Comment by gabrielliwerant** [comment]
Try experimenting with different stop propagation techniques and event handling: https://github.com/pixijs/pixijs/issues/10016.
